### ISSUE TITLE
Audible notification for presenter when a student raises their hand

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#1145]
Originally reported on Google Code with ID 390

```
When there are many people in a conference, such as over a hundred, the 
raise hand feature could be enhanced to provide the moderator/presenter 
with more information.

For example:

  # When a user raises their hand, BigBlueButton could display a dialog box 
asking them for the question.

  # When BigBlueButton displays users in the Web module, it could move 
users who raised their hand to the top of the list.

  # When the presenter or moderator moves their mouse of the user's name 
who raised their hand, a tool-tip could occur that shows their question.



```

Reported by `ffdixon` on 2010-03-07 14:54:29

**Comment by ffdixon** [comment]
Let's start with this: when a student raises their hand, the presenter should hear an audio message.

**Comment by ux-jules** [comment]
A recommendation:  give the teacher the ability to mute the audible and adjust the volume. I can see it being a disrupting in the middle of a sentence.

**Comment by ffdixon** [comment]
We are half-way in the HTML5 client as we can already notify the user when there is a new chat message

![image](https://user-images.githubusercontent.com/290351/68110609-516dca80-febb-11e9-9059-a151bf82cccd.png)

We need to add similar notification for raise hand.

**Comment by lnxbil** [comment]
Besides having an audio and visual information, It would also be very great if the "hand raise" feature would be a button at the bottom of the page. Raising a hand is essential and not a "normal status".

**Comment by YoniMeidan** [comment]
> 
> 
> Besides having an audio and visual information, It would also be very great if the "hand raise" feature would be a button at the bottom of the page. Raising a hand is essential and not a "normal status".

This is very important for us too. Including the notification to the presenter.

**Comment by german2209** [comment]
> Besides having an audio and visual information, It would also be very great if the "hand raise" feature would be a button at the bottom of the page. Raising a hand is essential and not a "normal status".

I agree, a button for "hand rise" more visible would be great instead of changing status.

**Comment by mtsonline** [comment]
so let's hope someone has the time to impliment this soon.

Really looking forward to this feature.

Thanks in advance for the effort.

**Comment by ffdixon** [comment]
We're making progress. Here's a look at the design for the new notifications tab for BigBlueButton 2.3 (next release).

![image](https://user-images.githubusercontent.com/290351/80540648-b3ddea00-8977-11ea-9262-0986cd75c50b.png)

**Comment by iSamof** [comment]
There are also some good suggestions about the same issue here: https://github.com/bigbluebutton/bigbluebutton/issues/9072

**Comment by joterini** [comment]
Hi,

Im curious if there will be also notofication when someone wait for acceptation?
It will be some implementation of web push? So if You are temporary working in other program, text editor for example, You will also receive notification on desktop.

GJ

**Comment by hananaq** [comment]
> We're making progress. Here's a look at the design for the new notifications tab for BigBlueButton 2.3 (next release).
> 
> ![image](https://user-images.githubusercontent.com/290351/80540648-b3ddea00-8977-11ea-9262-0986cd75c50b.png)

when will it be released ? any time soon ?

**Comment by ricubuntu** [comment]
> We are half-way in the HTML5 client as we can already notify the user when there is a new chat message
> 
> ![image](https://user-images.githubusercontent.com/290351/68110609-516dca80-febb-11e9-9059-a151bf82cccd.png)
> 
> We need to add similar notification for raise hand.

Hello Fred!
Is there any update about the rise hand notification?
Looking forward for your reply!
Thank you very much!
Riccardo

**Comment by paraviz02** [comment]
@ffdixon and the team/community —

Along with this excellent suggestion, it would be great to also have an audible/visual notice for the participant when their “hand raised” gets to the top of the list and a moderator is ready for them. We notice that the students are not always paying attention to the screen, but to an assignment on paper and it is difficult to make them aware that the moderator is ready to take their question.

Maybe this has already been suggested?

**Comment by antobinary** [comment]
Already addressed on 2.3-alpha6 or earlier (see the enabled toggles on the bottom row) Screenshot taken from test23.bigbluebutton.org
![Screenshot from 2021-02-08 16-40-26](https://user-images.githubusercontent.com/6312397/107284891-67f2c200-6a2c-11eb-9a32-8a9f0cc54981.png)
